### PR TITLE
feat: upgrade stitches version to 1.3.1-1

### PR DESCRIPTION
**Pull request by hdz-666**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/roomkit-react/src/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const StyledAvatar = styled('div', {
   ...flexCenter,
   color: '$colors$on_primary_high',
   fontFamily: '$sans',
-  aspectRatio: 1,
+  aspectRatio: '1 / 1',
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Avatar/Avatar.tsx:16]: Is this needed? There are lots of places where we are giving numeric values. Will all of them have to be replaced?

Comment (raviteja83) [reply packages/roomkit-react/src/Avatar/Avatar.tsx:16]: just '1' should be enough. was there a type error @hdz-666 ?

Comment (raviteja83) [reply packages/roomkit-react/src/Avatar/Avatar.tsx:16]: @hdz-666 ??

Comment (hdz-666) [reply packages/roomkit-react/src/Avatar/Avatar.tsx:16]: it was throwing error for 1/1

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Avatar/Avatar.tsx:16]: It should work with just '1'. Can you update other places where we use numbers?

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**